### ISSUE TITLE
Adding "Tabbed Card" component to Storybook 

### ISSUE THREAD

**Issue by katz95** [spotify/backstage#410]
Would love if someone could help create this "Tabbed Card" component to add to our Storybook (storybook.backstage.io)! 🙏

**Usage**
The tabbed card is one of the most basic UX components that will exist in Backstage. Its purpose is to provide informational content to users, while also giving them the option to switch contexts within that card. The distinction between the functionality of the "Tabbed Card" and the ["Pivot Card"](https://github.com/spotify/backstage/issues/401) is that the Tabbed card allows for more transparency into the various options, and also works better when there's a limited number of options. We recommend keeping the number of options under 5. There's the option of adding a footer that includes a link to another page/site/etc.

![Tabbed Card](https://user-images.githubusercontent.com/61153904/77890060-24a5bf80-726f-11ea-8196-24ea18623926.png)

 **Specs**
![Tabbed Card Specs](https://user-images.githubusercontent.com/61153904/77890072-2a030a00-726f-11ea-97e1-6c9deb2e259b.png)

**Comment by chris-nolan** [comment]
Hey, really like the project and keen to get involved. This looked like as good a place as any to get started! Had a couple of questions though:

1. Should a new Card be based off the existing InfoCard? It seems like things like the bottom link and card actions should be available and consistent across all Card types.
2. If we go with InfoCard then what is the state of that component? It isn't in TS so I guess worth waiting for #407 to deliver that. Plus we would need some functionality change - at least a way of getting in to the card header, I guess either pulling out the header as a component with its own children or passing in some elements that could be inserted between the BoldHeader and the Divider?
3. How to handle the Tab content? Following the material-ui pattern (https://material-ui.com/components/tabs/) seems reasonable, in which case we could go for something like (I'm not keen on the naming below btw, so alternatives welcome :) ):

```
<TabbedCard>
    <CardHeader title="Tabbed Card" divider>
        <CardTabs value={value} onChange={handleTabChange}>
            <CardTab label="Item One" />
            <CardTab label="Item Two" />
        </CardTabs>
    </CardHeader>
    <CardContent>
        <CardTabPanel value={value} index={0}>
            Item One
        </CardTab>
        <CardTabPanel value={value} index={1}>
            Item Two
        </CardTab>
    </CardContent>
</TabbedCard>
```
I won't commit to taking this on just yet, but hopefully can get a bit of direction and over the next few days I can have a stab at it

**Comment by katz95** [comment]
Awesome questions, and I appreciate your enthusiasm!! @chris-nolan 🥳

1. Yes, these two components should be consistent in the footer actions
2. Hmm I'm not sure how to best answer that rn, lemme see if i can find someone who can address that better than I can
3. Yes, following the material-ui pattern would be great 🙏

**Comment by hooloovooo** [comment]
I'm already working on this one. It's based off of the #440 branch. 
I'm trying to figure out a lot of these things as well. Going with your suggestion would give us a veeery thin layer on top of material-ui that mostly contains styling. It might be the way to go though.

**Comment by katz95** [comment]
ahhh okay let me assign you @hooloovooo 👍 do you know how to address question no. 2? if i'm interpreting it correctly it seems more from the dev perspective haha

**Comment by hooloovooo** [comment]
@katz95 Yup it's dev stuff. I think i've got some ideas on how to do that, and hopefully people will have opinions when i send in a PR as well. :)

**Comment by stefanalund** [comment]
@katz95 maybe you have another similar task for @chris-nolan to pick up?

**Comment by stefanalund** [comment]
Fixed #464
